clusters/build-clusters/common_except_app.ci/local_imagestreams.yaml

```diff
@@ -0,0 +1,62 @@
+# This file contains the image streams that
+# have to be existing on each build farms b01 ...
+# https://issues.redhat.com/browse/DPTP-3998
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: ci-operator
+  namespace: ci
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - from:
+      kind: DockerImage
+      name: quay.io/openshift/ci:ci_ci-operator_latest
+    importPolicy:
+      importMode: PreserveOriginal
+      scheduled: true
+    name: latest
+    referencePolicy:
+      type: Local
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: managed-clonerefs
+  namespace: ci
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - from:
+      kind: DockerImage
+      # We will use the image in the QCI when it is pushed there
+      # https://github.com/openshift/release/blob/master/clusters/build-clusters/build10/supplemental-ci-images/001_managed-clonerefs_mabc.yaml
+      name: registry.openshift.org/ci/managed-clonerefs:latest
```

Comment:
This image can live in quay as well, if you change the mabc to push it.

Reply:
I want to keep my task simple.